### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by golmansax**
Small readme change and a few code updates that are needed to get the server to run.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ This app supports 2 payment methods:
     donors give more than $1 if they like a day's nonprofit, let a donor
     cancel a $1 donation if they don't like that day's choice, etc)
   * overhead: you'll need to be a 501c3 to offer tax deductions to US donors, but their dashboard is very useful for that accounting if you are
-  * fees: $0.29 * donation amount + $0.30
+  * fees: $0.029 * donation amount + $0.30
```
Comment (tiegz) [inline README.md]: nice fix! Also % instead of $?

**Comment by golmansax** [comment]
@tiegz updated to 2.9%!

**Comment by tiegz** [comment]
Thanks @golmansax !

**Comment by tiegz** [comment]
@Branditoboss why $0.045?

**Pull request merged**